### PR TITLE
New WFP cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Cal csvs for the last WFP for Pioneer 16 - 13103-04
Vendor cals in Vault: Records > Platform Records > WFPs > Coastal_WFPs > 13103-04
OR Alfresco: Instrument & Platform Documents > Calibration and Repair > Coastal-Global Arrays > <instrument folder>

### CODE REVIEW

File: calibration/PARADK/CGINS-PARADK-20582__20210113.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20582,CC_dark_offset,1.3, source_file PARAD-K_QSP2200_SN_20582_Calibration_2021-01-13.pdf
+20582,CC_scale_wet,9.89E-18,
```
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20582__20210113.csv:3]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-01100__20210108.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1100,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_1100_Calibration_2021-01-08.zip > BBFL2-1100 Charsheets.pdf
+1100,CC_dark_counts_cdom,49,
+1100,CC_dark_counts_chlorophyll_a,48,
+1100,CC_dark_counts_volume_scatter,48,
+1100,CC_depolarization_ratio,0.039,
+1100,CC_measurement_wavelength,700,
+1100,CC_scale_factor_cdom,0.0907,
+1100,CC_scale_factor_chlorophyll_a,0.0121,
+1100,CC_scale_factor_volume_scatter,3.121E-06,
+1100,CC_scattering_angle,124,
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01100__20210108.csv:11]: Looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20210114.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2726,CC_frequency_offset,-8.305002e+002, source_file DOFST-K_SBE43F_SN_2726_Calibration_2021-01-14.zip > 2726.cal and SBE 43F O2726 SOCAdjusted 20Jan21.pdf
+43-2726,CC_oxygen_signal_slope,3.3281e-004, adjusted Soc value
+43-2726,CC_residual_temperature_correction_factor_a,-3.620252e-003,
+43-2726,CC_residual_temperature_correction_factor_b,1.795585e-004,
+43-2726,CC_residual_temperature_correction_factor_c,-2.603914e-006,
+43-2726,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20210114.csv:7]: Looks good

**Review by cdobs — APPROVED** [review]
All look good

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20210114.csv (hunk 3 above)
Comment (batryn) [reply calibration/DOFSTK/CGINS-DOFSTK-02726__20210114.csv:7]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-01100__20210108.csv (hunk 2 above)
Comment (batryn) [reply calibration/FLORTK/CGINS-FLORTK-01100__20210108.csv:11]: Looks good

File: calibration/PARADK/CGINS-PARADK-20582__20210113.csv (hunk 1 above)
Comment (batryn) [reply calibration/PARADK/CGINS-PARADK-20582__20210113.csv:3]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Pull request merged**